plugins/package-managers/cocoapods/src/main/kotlin/Lockfile.kt

```diff
@@ -58,7 +58,7 @@ internal fun String.parseLockfile(): Lockfile {
     val root = Yaml.default.parseToYamlNode(this).yamlMap
     val pods = root.get<YamlList>("PODS")?.items.orEmpty().map { it.toPod() }
 
-    val checkoutOptions = root.get<YamlMap>("CHECKOUT OPTIONS")?.entries.orEmpty().mapNotNull {
+    val checkoutOptions = root.get<YamlMap>("CHECKOUT OPTIONS")?.entries.orEmpty().map {
```

Comment:
Could this as well please be done before migration, as one of the first commits in this PR, to better explain that it is unrelated to be migration?